scripts/build/termux_step_setup_toolchain.sh

```diff
@@ -196,10 +196,11 @@ termux_setup_standalone_toolchain() {
 	# Remove <sys/sem.h> as it doesn't work for non-root.
 	# Remove <glob.h> as we currently provide it from libandroid-glob.
 	# Remove <iconv.h> as it's provided by libiconv.
+	# Remove <semaphore.h> as it's provided by libandroid-posix-semaphore.
 	# Remove <spawn.h> as it's only for future (later than android-27).
 	# Remove <zlib.h> and <zconf.h> as we build our own zlib.
 	# Remove unicode headers provided by libicu.
-	rm usr/include/{sys/{capability,shm,sem},{glob,iconv,spawn,zlib,zconf}}.h
+	rm usr/include/{sys/{capability,shm,sem},{glob,iconv,semaphore,spawn,zlib,zconf}}.h
```

Comment:
Are we sure moving this to a separate package won't break building any other packages, or are we going to shift all those packages to listing this one as a dependency eventually? @Grimler91, you may want to chime in.

Reply:
Can do some test builds and see if anything, or how much, that brakes

Reply:
Maybe we can add this patch into `libandroid-support` package rather than split into a sperated package. It would not break other package's dependency.